pandarallel/core.py

```diff
@@ -470,6 +474,7 @@ def initialize(
             if use_memory_fs
             else parallelize_with_pipe
         )
+        parallelize = partial(parallelize, single_bar=single_bar)
```

Comment:
IMO instead it would be better to supply the argument in lines 528 onward as it is currently done with the arguments of `parallelize`. @nalepae any opinion?